modules/tinymce/src/core/main/ts/keyboard/SpaceKey.ts

```diff
@@ -8,17 +8,22 @@
 import Editor from '../api/Editor';
 import { EditorEvent } from '../api/util/EventDispatcher';
 import VK from '../api/util/VK';
-import { fireFakeInputEvent } from './FakeInputEvents';
+import { fireFakeBeforeInputEvent, fireFakeInputEvent } from './FakeInputEvents';
 import * as InsertSpace from './InsertSpace';
 import * as MatchKeys from './MatchKeys';
 
 const executeKeydownOverride = (editor: Editor, evt: KeyboardEvent) => {
```

Comment:
This before input logic needs some tests as I didn't see any added for it.

- [ ] Add `beforeinput` tests for the space key override

Reply:
Added a few more tests. I kept them in separate files to try and avoid making one big filed that tested a bunch of unrelated functions, but if we prefer it in a different way do tell.